examples/lock-app/efr32/src/EventHandler.cpp

```diff
@@ -0,0 +1,206 @@
+/*
```

Comment:
Looks good but I do wonder if it wouldn't be simpler/fewer functions to have one command with a parameter for the event you want to simulate.
And based on that parameter a switch case would decide what actions is done on the cluster.

Reply:
I built it this way because 
1. Even though these events only have 1 additional parameter (AlarmCode, DoorState), other events added in the future may have multiple additional parameters.
2. It seemed nicer to the user to be able to input "lock-alarm", "door-state-change" rather than have to input the event code. But doing it this way I don't have anything to switch on, unless I put string literals in the switch

Reply:
the user would have to enter the event code.

the shell user would do something like
doorEvent lock-alarm
doorEvent state-change
etc. 
then you only have 1 shell command handler and different options to trigger an event.

Reply:
Removed duplicate function and replaced it with a switch case